examples/hif2a/hifa2_config.yaml

```diff
@@ -0,0 +1,15 @@
+---
+map-generation:
+  strategy: geometry
```

Comment:
Should we specify this further as `atom_mapping_strategy` or similar? It's foreseeable we would could elaborate on this to reference parameterized strategies for other tasks (e.g. an `edge_selection_strategy`).

Reply:
Resolved by @badisa in https://github.com/proteneer/timemachine/pull/368/commits/dde1d3b5fd571338c99187718fcf800628e55d27